### PR TITLE
Update: Replace arrow function with normal function in unit test

### PR DESCRIPTION
**Pull request by rmeena840**

### CODE REVIEW

**Comment by sczembor** [comment]
Looks good. Lets update change log. I also think we should add description to our guide about tests. Lets explain the reason for that and add link to mocha documentation

File: docs/guide/best-practices.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ layout: splash
 
 - Use [boilerplate-stateful-smart-contract](https://developer.algorand.org/docs/features/asc1/stateful/#boilerplate-stateful-smart-contract) as a template for new smart-contracts.
 - Use [zero address](https://developer.algorand.org/docs/features/accounts/#special-accounts) to prevent future updates or deletion of a smart-contract.
+- Use javascript normal functions instead of arrow function while writing unit tests. Please read more
```
Comment (sczembor) [inline docs/guide/best-practices.md:9]: Maybe we should encourage the user a bit more. `It is strongly recommended not to use arrow functions in tests due to the problem of scope of `this` ...  Use only regular function expressions. For more details go to`

File: CHANGELOG.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ Features, Bug Fixes, API Breaking, Deprecated, Infrastructure, Template Updates
 - Fix `txn AssetSender` should return zero address by default.
 - Add unit tests for all transaction types in runtime executeTx.
 - Add support loadLogic for *.teal program and SCParam.
+- Replace arrow functions with normal functions in unit tests.
```
Comment (sczembor) [inline CHANGELOG.md:22]:
```suggestion
- Replace arrow functions with normal functions in all unit test which is recommended by `mocha`
```

**Review by sczembor — APPROVED** [review]
Pre-approval. Please check the comments I left before merging

File: docs/guide/best-practices.md (hunk 1 above)
Comment (rmeena840) [reply docs/guide/best-practices.md]: Done

**Pull request merged**